group_vars/development

```diff
@@ -1,4 +1,6 @@
 mysql_root_password: devpw
+mailhog_binary_url: https://github.com/mailhog/MailHog/releases/download/v0.1.7/MailHog_linux_amd64
```

Comment:
This will not be needed once https://github.com/geerlingguy/ansible-role-mailhog/pull/6 is merged in.